src/autoscaler/eventgenerator/aggregator/metricPoller.go

```diff
@@ -104,17 +102,17 @@ func (m *MetricPoller) doAggregate(appId string, metricType string, metrics []*m
 		avgAppMetric = &model.AppMetric{
 			AppId:      appId,
 			MetricType: metricType,
-			Value:      0,
+			Value:      -1,
 			Unit:       "",
-			Timestamp:  0,
+			Timestamp:  time.Now().UnixNano(),
```

Comment:
This seems wrong.
We are walking over a set of metrics. The current code will assign a different timestamp to each. Logically, they should all have the same timestamp.